index.js

```diff
@@ -212,11 +224,11 @@ function updateForToken(token, data={}) {
             y: (data.y ?? token.y) + ((token.parent?.dimensions?.size / 2) ?? 0),
         };
         canvas.worldExplorer.reveal(center);
-    } 
+    }
 }
 
-const refreshThrottled = foundry.utils.throttle(() => {
-    if (canvas.worldExplorer?.settings.revealRadius) {
-        canvas.worldExplorer.refreshMask();
+const refreshThrottled = foundry.utils.throttle((force) => {
```

Comment:
This may not work the way you intended. Something to take note is that within a throttle window while a function is pending, all later functions are dropped. So if a refreshThrottled() is pending, and you call refreeshThrottled(true), it'll be dropped.